src/buffer/out/textBuffer.cpp

```diff
@@ -944,6 +944,115 @@ Microsoft::Console::Render::IRenderTarget& TextBuffer::GetRenderTarget() noexcep
     return _renderTarget;
 }
 
+// Method Description:
+// - Get the COORD for the beginning of the word you are on
+// Arguments:
+// - target - a COORD on the word you are currently on
+// - wordDelimiters - what characters are we considering for the separation of words
+// - includeCharacterRun - include the character run located at the beginning of the word
+// Return Value:
+// - The COORD for the first character on the "word"  (inclusive)
+const COORD TextBuffer::GetWordStart(const COORD target, const std::wstring wordDelimiters, bool includeCharacterRun) const

```

Comment:
Should this be a `const std::wstring&`, or a `string_view`? I don't think we should need to copy this string each method call